modules/dnn/src/int8layers/convolution_layer.cpp

```diff
@@ -969,6 +971,13 @@ class ConvolutionLayerInt8Impl CV_FINAL : public BaseConvolutionLayerInt8Impl
                                     stride_h, stride_w, dilation_h, dilation_w, pad_t, pad_l,
                                     biasptr, multptr, inptr_, height, width, outptr_, out_d, outH, outW, inpZp, outZp);
                             else
+                        #endif
+                        #if CV_RVP052
+                            if(useRVP052)
```

Comment:
In `fully_connected_layer.cpp` this is absolutely right. But in `convolution_layer.cpp`, `useRVP052` is not always the same as `CV_RVP052`, because of `line 769` `p.useRVP052 = CV_RVP052 && isConv2D;` introducing a little difference.
So change this boolean flag into `isConv2D` might be better.